documentation/tut/toy-example-from-scratch.rst

```diff
@@ -322,5 +329,27 @@ We can also look at the charging schedule in the `FlexMeasures UI <http://localh
 
 Recall that we only asked for a 12 hour schedule here. We started our schedule *after* the high price peak (at 5am) and it also had to end *before* the second price peak fully realised (at 9pm). Our scheduler didn't have many opportunities to optimize, but it found some. For instance, it does buy at the lowest price (around 3pm) and sells it off when prices start rising again (around 6pm).
 
+.. note:: In case we want to delete toy-account and add it again to run the tutorial, it is recommended to delete the database ``flexmeasures-db`` that we have created for this tutorial because on the same database we will get different IDs for the sensors. Follow the following commands.
+
+.. code-block:: console
+
+    # logged in as the postgres superuser
+    $ sudo -u postgres psql
+    # delete the database
+    $ DROP DATABASE flexmeasures-db
+    # create new database
+    $ CREATE DATABASE flexmeasures-db
+    # now connect to database and create extensions
+    $ \connect flexmeasures-db
+    $ CREATE EXTENSION cube;
+    $ CREATE EXTENSION earthdistance;
+    $ exit
+
+Then we will the get database structure with
+
+.. code-block:: console
+
+    $ flexmeasures db migrate
```

Comment:
Actually, as under "On your PC" I believe just thee `upgrade` command should suffice right?
In any case, we need to be  consistent.

Reply:
yes, you are right, only upgrade works fine.

Reply:
FYI: the `flexmeasures db migrate` command is an alias for the `flexmeasures db revision` command, and either one just creates a new revision file, which was never your intention in this PR. :)